blockstore/apps/api/routers.py

```diff
@@ -36,11 +36,9 @@ def get_lookup_regex(self, viewset, lookup_prefix=''):
             lookup_url_kwargs = getattr(viewset, 'lookup_url_kwargs', lookup_fields)
             lookup_value_regexes = getattr(viewset, 'lookup_value_regexes', len(lookup_fields) * ['[^/.]+'])
             return ','.join([
-                '(?P<{lookup_prefix}{lookup_url_kwarg}>{lookup_value_regex})'.format(
-                    lookup_prefix=lookup_prefix,
-                    lookup_url_kwarg=lookup_url_kwarg,
-                    lookup_value_regex=lookup_value_regex,
-                ) for (lookup_url_kwarg, lookup_value_regex) in zip(lookup_url_kwargs, lookup_value_regexes)
+                f'(?P<{lookup_prefix}{lookup_url_kwarg}>{lookup_value_regex})' for (
+                    lookup_url_kwarg, lookup_value_regex
+                ) in zip(lookup_url_kwargs, lookup_value_regexes)
```

Comment:
There are a few conversions to f-strings because of quality issues raised by pylint version upgrade.